pkg/controller/migcluster/migcluster_controller.go

```diff
@@ -97,12 +99,14 @@ var _ reconcile.Reconciler = &ReconcileMigCluster{}
 type ReconcileMigCluster struct {
 	k8sclient.Client
 	scheme     *runtime.Scheme
+	record.EventRecorder
```

Comment:
I would put this one line above so all embedded types are together and then have a blank line:

```
    k8sclient.Client
    record.EventRecorder

    scheme *runtime.Scheme
```